docs/commands/rpc-protocol.md

```diff
@@ -2787,9 +2787,14 @@ A valid Nano account. If provided and `use_peers` is set to `true`, this informa
 
 **Optional "version"**
 
-_version 21.0+_
+_version 21.0+_  
 Work version string. Currently "work_1" is the default and only valid option.
 
+**Optional "block"**
+
+_version 21.0+_  
+A valid Nano block (string or JSON). Using the optional `json_block` is recommended. If provided and no `difficulty` or `multiplier` are set, is used to calculate work generation difficulty for given block from node ledger.
```

Comment:
Should we use a similar description to RPC block_create optional "difficulty"?

Reply:
We have "If difficulty and work values are both not given, RPC processor tries to calculate difficulty for work generation based on ledger data: epoch from previous block or from link for receive subtype; block subtype from previous block balance."

Reply:
```suggestion
A valid Nano block (string or JSON). Using the optional `json_block` is recommended. If provided and `difficulty` or `multiplier` are both not given, RPC processor tries to calculate the approprriate difficulty threshold based on ledger data.
```

Like this maybe?

Could you also fix the lack of _newline_ in `block_create` ? See image
![image](https://user-images.githubusercontent.com/12448106/82983208-9c7e3500-9fe7-11ea-8dcd-ac9316b29756.png)